packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1472,10 +1496,12 @@ final copyPackagePathOption = ScriptPopupMenuOption(
 final copyFilePathOption = ScriptPopupMenuOption(
   label: 'Copy file path',
   icon: Icons.content_copy,
-  onSelected: (_, controller) async {
-    return Clipboard.setData(
-      ClipboardData(text: await fetchScriptLocationFullFilePath(controller)),
-    );
+  onSelected: (_, controller) {
+    unawaited(() async {
+      Clipboard.setData(
```

Comment:
this is confusing... why does Clipboard.setData need to be wrapped as an unawaited async callback?